Source/Urho3D/UI/UIEvents.h

```diff
@@ -56,8 +56,8 @@ URHO3D_EVENT(E_UIMOUSEDOUBLECLICK, UIMouseDoubleClick)
     URHO3D_PARAM(P_ELEMENT, Element);              // UIElement pointer
     URHO3D_PARAM(P_X, X);                          // int
     URHO3D_PARAM(P_Y, Y);                          // int
-	URHO3D_PARAM(P_XBegin, XBegin);                // int
-	URHO3D_PARAM(P_YBegin, YBegin);                // int
+    URHO3D_PARAM(P_XBEGIN, XBEGIN);                // int

```

Comment:
Sorry for confusing. I mean `(P_XBEGIN, XBegin)`

Reply:
Ah yes - I see that now.